src/Handler/CanNotInvokeHandler.php

```diff
@@ -39,12 +35,15 @@ public static function forCommand($command, string $reason)
         return $exception;
     }
 
+    private function __construct(string $message = '', $code = 0, Throwable $previous = null)
+    {
+        parent::__construct($message, $code, $previous);
+    }
+
```

Comment:
I like the intent here but I prefer to leave the constructor open so other folks can potentially use it down stream. Maybe they'd like to format something entirely different here but still use a Tactician level exception. :)

Reply:
Look at `command` property. And getter. After one constructor instance will be broken.

Reply:
Yeah, there is no way to set it short of doing something dumb like a protected setter. Alright, cool, let's keep it :)